(/social-connections) align with dashboard terminology; use partials (#1855)
clerk/clerk-docs: docs/authentication/social-connections/huggingface.mdx

> To configure your production instance, follow these steps:

Author: the line above needs be to updated to

"In _production instances_, you must provide custom credentials."

Reviewer: https://github.com/clerk/clerk-docs/pull/1855/commits/6f0f393e8e21e5ec1fe13e30d60e16f8e598c75e

<Steps>
  ### Enable Hugging Face as a social connection

  1. In the Clerk Dashboard, navigate to the [**SSO connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
  1. Select the **Add connection** button and select **For all users**.
  1. In the **Choose provider** dropdown, select **Hugging Face**.
  1. Ensure that both **Enable for sign-up and sign-in** and **Use custom credentials** are toggled on.
  1. Save the **Redirect URL** somewhere secure. Keep this modal and page open.

  ### Create a Hugging Face Connected App

>   1. Navigate to [Hugging Face](https://huggingface.co/) and sign in to your account.

Author: ```suggestion
  1. In the top-right of [Hugging Face](https://huggingface.co/), select your avatar and select **Settings**.
```

>   1. In the top-right, select your avatar and select **Settings**.

Author: ```suggestion
```

>   1. In the sidebar, select **Connected Apps**.

Author: ```suggestion
  1. In the left sidebar, select **Connected Apps**.
```

  1. Under **Developer Applications**, select **Create App**.
  1. Complete the form as follows:
     - **Application Name**: Name the application whatever you'd like.
     - **Homepage URL**: Enter the homepage URL of your app.
     - **Logo URL**: Enter the URL of the logo for your app.
     - **Token Expiration**: Set the duration that the token will be valid.
     - **Scopes**: Select the scopes that your app requires. Ensure that **openid**, **profile**, and **email** are included.

>      - **Redirect URLs**: Paste the **Redirect URL** value you saved from the Clerk Dashboard.

Author: this entire "Complete the form as follows:" step can be more concise:

1. Fill in the necessary information. Under **Scopes**, select the scopes that your app requires. At minimum, select **openid**, **profile**, and **email**. Under **Redirect URLs**, paste the **Redirect URL** value you saved from the Clerk Dashboard.

Reviewer: https://github.com/clerk/clerk-docs/pull/1855/commits/6f0f393e8e21e5ec1fe13e30d60e16f8e598c75e

>   1. Select **Create application**. The page should refresh and display the **Client ID** and **App Secret** under **Application Name**.

Author: ```suggestion
  1. Select **Create application**. The page should refresh and display the **Client ID** and **App Secret**. Save these values somewhere secure.
```

>   1. Save the **Client ID** and **App Secret** somewhere secure.

Author: ```suggestion
```
